test/unit/test-form.js

```diff
@@ -585,3 +590,110 @@ describes.realWin('isFieldDefault', {}, env => {
     });
   });
 });
+
+describes.fakeWin('isFieldEmpty', {}, env => {
+  let doc;
+
+  beforeEach(() => {
+    doc = env.win.document;
+  });
+
+  describe('text field', () => {
```

Comment:
nit: rearrange tests in the order similar to your implementation. 

- "input element" -> "type checkbox" -> empty/not empty
- "input element" -> "type radio" -> empty/not empty
- "input element" -> "type others" -> empty/not empty
- "textarea" -> empty/not empty
- "select"
-  "default"